docs/applications/containers/kubernetes/static-site-linode-kubernetes-engine/index.md

```diff
@@ -544,24 +609,32 @@ kubernetes&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Clus
 static-site-service&nbsp;&nbsp;LoadBalancer&nbsp;&nbsp;10.128.99.240&nbsp;&nbsp;192.0.2.1&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;80:32648/TCP&nbsp;&nbsp;&nbsp;100m
 {{</ output >}}
 
-    Note the external IP address of the Service you created. This is the IP address of the NodeBalancer, and you can use it to view your static site. In the above example, the IP address is `192.0.2.1`. Navigate to the external IP address in the browser of your choice to view your static site.
+1.  Note the external IP address of the Service you created. This is the IP address of the NodeBalancer, and you can use it to view your static site.
+
+1.  In the above example, the IP address is `192.0.2.1`. Navigate to the external IP address in the browser of your choice to view your static site. You should see the same content as when you tested your Docker Image on your workstation.
+
+## Next Steps
```

Comment:
Adding a next steps section that directs users to setting up DNS if they want to

Reply:
If we ever write a guide that explains Linode's external DNS plugin for kubernetes, we could link to that too/instead